frontend/src/metabase/dashboard/hooks/use-is-parameter-panel-sticky.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useRef, useState } from "react";
+
+export function useIsParameterPanelSticky() {
```

Comment:
It was used for embedded preview before to control the parameters panel sticky state.
Now it is used on the regular dashboard as well for the same purpose instead of an old helper.